packages/cli/src/generators/templates/authTests/basic.test.ts

```diff
@@ -0,0 +1,42 @@
+import { describe, expect, it } from 'vitest';
+import zapier from 'zapier-platform-core';
+
+import App from '../index.js';
```

Comment:
Similar to https://github.com/zapier/zapier-platform/pull/1067#discussion_r2213955287. And applies to other auth test templates for typescript.